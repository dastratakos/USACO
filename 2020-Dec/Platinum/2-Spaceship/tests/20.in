60 60 60
100001010011111010011000101101000011001101000110111001100111
101101010100101011110101100110001111100001111111101101000001
111011110111101111111110000011111010111010111101100010111010
101111111101111110011111010011011101110101111110111010111100
100001111111101111010111111101110111100111101101110011011111
101101011001110010010111011100000001000111010111000001011011
111100110111111011011000010011010100011011001110010101010100
011101101110111011001110010001110001011111001101111111100111
111011011110101101110100000110111111110111111110001100110001
111110010100100111100010001111100101001100101111000111111111
111101000011000111111101101111010111010111110010101111011110
101100111110110101101011010011110111111111010111011101111101
001110100110111110011110100111111111011110110101011001110001
111000100110001111111111101111111001010111010011101001111111
111111101101111101110111010011111100011110111111010101110111
100101011101111110101111111111100101111110110001000100111101
100111110010001101101011111111011100110111101110101101110110
100011101110111001111101111011111111111101100011001111011001
110001101111111011110101111011111111100111101111011001101111
101101111001010111101111010111011111001101101111001110001110
011010101110011111110011010001110111001111101111110110100110
111111111111001110101111110010110111101111110100010000011011
111111010101101011011111011100110010010001111111000111011101
011101100101101101111110011111111111111111001010101111111101
001111111110111110011100001110011111101110001001101111111011
010001011001111101011101111111101101110111111101000111010111
101100111111001111101110011101010010111110010101110101101011
111101011111101101100101111110010111011101111011011011111010
111010111001001000101111001001111111111111100011111100011110
001011111011111111111110010000010111111010001111110100010011
101111111000111101001101100110110110100111011110111101010111
110101110111000111010011000111111111001111111110010110011011
100010100110110010001011101110100000111110001110101111101010
111011101111111101110111011111111110101011010000000110000111
001011010111111110111110111111111000101111000110011001001100
111111001111110001101011111010100111000011010011011110101011
010111100100111010100101111111010110001011011010111000001010
110011101110111011101101110101110111100111010011010110111001
011101011101100011110011101000111111101101011111111111110101
010100110010010011001100111100001100010101100110101110011011
011111110101111100110100101011110100010010111111011111011101
101111111111101110011110111101001110111110011111000110010000
100111110110111011101110011011100110110111001010111101111111
111000111010101010110111011011111011111101100010110010110101
000101000011010001111100101011111111101111110000101111110110
110011111111101111111111011100011011111111011101110111110110
001110011111111001011111111011000101100111111111111011101110
010101000111100111110110111010110011011101101010101101011101
010011110111111110100110110100101100111110011110010010111010
011111100111101011111111010111011100110011011111101111101110
111011011111001111110011101111101100101000011110111000010100
111101011111111101100110111001110010111011111100110010110110
111011100111010010010111011111011011011111111111100111111111
111111100011110111111011001111110100111110111110110001111110
101111101111011110111101011111110111100110001110111101010111
101111011011011010101111111010001101001101010111111011101100
110101100111111111011111111110111101001100111111000110011000
110101111111100011101010011111111111101010001111001111010011
101101111111101110101111001110111100011010111111101111110011
111111111111001111110100101100111010111111110100110101011110
1 13 4 46
3 55 3 14
4 49 1 23
2 49 2 50
3 3 3 34
1 26 4 2
2 8 4 11
55 23 3 37
2 17 7 51
49 57 17 32
2 40 4 46
4 59 30 19
5 55 2 7
58 57 3 59
5 41 1 20
43 26 3 6
44 58 4 17
2 6 1 33
21 15 52 16
4 50 1 45
3 1 4 15
2 27 4 6
4 12 3 32
8 2 2 52
60 11 13 38
1 27 1 12
4 3 4 34
1 37 4 38
1 11 45 56
3 21 52 3
4 7 5 2
1 29 2 10
2 55 4 44
2 45 13 14
3 3 4 47
1 57 5 27
2 38 44 51
3 33 1 38
1 28 3 32
42 6 1 12
2 26 1 18
1 36 59 42
4 36 3 56
1 55 3 4
5 38 1 9
1 29 10 32
2 5 1 57
5 24 35 25
4 59 2 27
3 15 2 8
4 14 3 35
3 29 38 47
2 15 5 18
2 29 1 59
2 5 12 37
5 60 2 14
1 37 2 8
1 12 2 2
15 38 53 9
5 22 5 22
