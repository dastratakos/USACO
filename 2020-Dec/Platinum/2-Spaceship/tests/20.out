54891655
292979577
313100228
829867659
308710655
899101442
853325052
612822107
795029427
249728493
276429687
264391074
141837729
621365716
278230449
155446827
405613077
729542340
968307680
678278950
665479592
419541794
739372011
180952215
903040288
926976837
915150836
282826541
875601250
401417
485446044
67344658
20331085
192436843
129153546
296734504
181292783
166082446
739969723
767950418
175287905
120003152
169731236
849604287
586852181
979402116
405315055
664762986
499971737
537707283
483129952
351813256
528927142
312824565
127446996
873893027
642669633
200739860
704063092
341486443
